spec/factories/users.rb

```diff
@@ -0,0 +1,10 @@
+FactoryBot.define do
+  factory :user do
+    name { "John Doe" }
+    email { "john.doe@example.com" }
+    password { "password123" }
+    phone { "12345678901" }
+    user_name { "johndoe" }
+    role { :user }
+  end
+end
```

Comment:
You can use `faker` gem for dummy values like these. Then you don't have to think of different values for the attributes every time you create a user instance.